paypalscript.py

```diff
@@ -27,6 +29,16 @@ def read(self):
         except FileNotFoundError:
             exit(f"File { self.in_file } not found")
 
+    def validate(self, validation_date):
+        validated_data = []
+        for row in self.data:
+            date_time_obj = datetime.strptime(row["Date"] + " " + row["Time"], "%m/%d/%Y %H:%M:%S")
+            if date_time_obj <= validation_date:
+                print(f'Ignoring Row for Date ' + row["Date"] + ' ' + row["Time"] + ' for contributor ' + row["Name"])
```

Comment:
Use f-string properly:

print(f'Ignoring Row for Date { row['Date'] } { row['Time'] } for contributor { row['Name'] }')